README.md

```diff
@@ -63,9 +63,27 @@ puzzle --mode gemini view -i path/to/gemini_database.db
 Puzzle uses the ped file to show more information in family view and in variant calls:
 
 ```
-puzzle view -i tests/fixtures/hapmap.vcf --family_file/-f tests/fixtures/hapmap.vcf
+puzzle view -i tests/fixtures/hapmap.vcf --family_file/-f tests/fixtures/hapmap.ped
 ```
 
+## Establish a persistent local database
+
+Puzzle can establish a persistent local database, so you can keep work between sessions.
+This enables other key features, such as gene lists for filtering and variant triage.
+First, you need to initialize a local database. Then one or more cases can be loaded and 
+are then ready for viewing as soon as the web server is started.
+
+```
+puzzle init --root tests/fixtures
```

Comment:
Nice proposal, I see the benefits of integration! However, I see several drawbacks on this:

1. You'd be introducing a hard GEMINI dependency for puzzle. Kind of forcing the user to have not only GEMINI, but a GEMINI db in place.
2. As of by now, you can't add or remove variants form a GEMINI database, right? You've to reconstruct the whole thing. What if the user needs to do that? All the `puzzle_` table will be gone

Also, the point with Puzzle is to be portable and independent, allowing users to do simple things like visualize a VCF in a nice way. I think that what you propose would make this task less straightforward.

Just my 5 cents :-) What do the main devels. think?